composer.json

```diff
@@ -8,7 +8,9 @@
         "ext-json": "*",
         "guzzlehttp/guzzle": "^6.5",
         "psr/http-client": "^1.0",
-        "php-http/guzzle6-adapter": "^2.0"
+        "php-http/guzzle6-adapter": "^2.0",
+        "endclothing/prometheus_client_php": "^1.0",
+        "webmozart/assert": "^1.9"
```

Comment:
I think that we should keep the library as slim as possible and avoid as much as we can to add extra libraries. I understand the fact that the webmozart/assert library makes the code cleaner, but in the end, this is a library that may be used in a lot of projects and we will force all users to integrate the webmozart/assert in their projects.

As I saw below, the asserts made with webmozart are not very complex, and the code will remain elegant even if you switch to plain PHP.

Reply:
Assert library removed